### PR TITLE
fix embedded gists

### LINKED ISSUE #658: old embedded gists?
switch to https://gohugo.io/content-management/shortcodes/#gist

cc @sckott

### PR DESCRIPTION
**Pull request by maelle**
Fix  #658

I looked for "<script".

Example of embedded gist https://deploy-preview-660--ropensci.netlify.com/blog/2012/11/26/is-invasive/

### CODE REVIEW

File: content/blog/2017-11-21-ochRe.md [hunk 1]
```diff
@@ -22,8 +22,6 @@ tags:
   - unconf
 ---
 
-<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (sckott) [inline content/blog/2017-11-21-ochRe.md:25]: there's i think 3 files where `https://platform.twitter.com/widgets.js` is removed, not sure what that does or if its needed still

Comment (maelle) [reply content/blog/2017-11-21-ochRe.md:25]: No longer needed, and I caught it whilst looking for embedded gists.

Comment (maelle) [reply content/blog/2017-11-21-ochRe.md:25]: Sorry for the lack of context

**Comment by maelle** [comment]
Thx

**Pull request merged**